ds3-sdk/src/main/java/com/spectralogic/ds3client/helpers/strategy/transferstrategy/AbstractTransferStrategy.java

```diff
@@ -223,6 +244,19 @@ private Objects firstChunk() {
 
     @Override
     public void close() throws IOException {
+        canceled = true;
         executorService.shutdown();
     }
+
+    @Override
+    public void cancel() throws IOException {
+        close();
+        final UUID jobId = masterObjectList.getJobId();
+        ds3Client.cancelJobSpectraS3(new CancelJobSpectraS3Request(jobId));
+        eventDispatcher.emitCanceledEvent(jobId);
+    }
+
+    private interface Predicate {
```

Comment:
Do we need to define a new Predicate interface?  I thought Java already had one: https://docs.oracle.com/javase/8/docs/api/java/util/function/Predicate.html